spec/01-unit/019-hooks_spec.lua

```diff
@@ -0,0 +1,88 @@
+local hooks = require "kong.hooks"
+
+local c = 0
+local function s()
+  c=c+1
+  return tostring(c)
```

Comment:
Is the return of this function (the value of c) used on any test? Couldn't `h` just be initialized to `function() return true end`?

Reply:
`h` is set before each test to the name of the current test's hook

Reply:
so the return is used